system-int-test/test/cmr/system_int_test/ingest/granule_ingest_test.clj

```diff
@@ -532,6 +532,22 @@
          (is (= 201 status))
          (is (= nil errors))))))
 
+(deftest old-delete-time-granule-ingest-test
+  (testing "DeleteTime in past results in validation error"
+    (let [collection (d/ingest-umm-spec-collection
+                      "PROV1"
+                      (data-umm-c/collection 1 {}))
+          granule (-> (dg/granule-with-umm-spec-collection
+                       collection
+                       (:concept-id collection)
+                       {:data-provider-timestamps {:delete-time "2000-01-01T00:00:00Z"}})
+                      (d/item->concept :umm-json)
+                      ingest/ingest-concept)]
```

Comment:
Separate the ingest step outside of the granule creation to make the test easier to read, and the ingest status and errors more obvious.